src/crimson/common/errorator.h

```diff
@@ -964,16 +964,6 @@ struct errorator {
   public:
     using type = ::crimson::errorator<AllowedErrors...>::future<ValueT>;
 
-    template <class Func, class... Args>
```

Comment:
Typo in commit msg – `s/futuruze/futurize/`.

Reply:
fixed